src/OpenLoco/Windows/Tutorial.cpp

```diff
@@ -0,0 +1,87 @@
+#include "../Tutorial.h"
+#include "../Graphics/Colour.h"
+#include "../Graphics/Gfx.h"
+#include "../Localisation/FormatArguments.hpp"
+#include "../Objects/InterfaceSkinObject.h"
+#include "../Objects/ObjectManager.h"
+#include "../OpenLoco.h"
+#include "../Ui.h"
+#include "../Ui/WindowManager.h"
+
+#include <array>
+
+namespace OpenLoco::Ui::Windows::Tutorial
+{
+    enum Widx
+    {
+        frame,
+    };
+
+    constexpr Gfx::ui_size_t windowSize = { 140, 29 };
+
+    widget_t widgets[] = {
+        makeWidget({ 0, 0 }, windowSize, widget_type::wt_3, 0),
+        widgetEnd(),
+    };
+
+    static window_event_list _events;
+
+    static void initEvents();
+
+    // 0x00438CAE
+    window* open()
+    {
+        initEvents();
+
+        auto window = WindowManager::createWindow(
```

Comment:
is this not a createWindowCentered?

Reply:
It's only horizontally centred, at the bottom of the screen. `createWindowCentred` centres on both axes.